src/pages/Main/Gbnow/Gbnow.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import './Gbnow.scss';
+import { Link } from 'react-router-dom';
+class Gbnow extends React.Component {
```

Comment:
- 다음의 import 순서 지켜주세요! 
- React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)
- 그리고 import 구문과 class 사이는 한칸 띄워주세요!
```suggestion
import React from 'react';
import { Link } from 'react-router-dom';
import './Gbnow.scss';

class Gbnow extends React.Component {
```